demonstrations/qsim_supremacy.py

```diff
@@ -2,11 +2,11 @@
 
 .. _quantum_neural_net:
 
-Quantum supremacy using qsim
-============================
+Beyond classical computing with qsim
+====================================
 
 .. meta::
-    :property="og:description": Discover quantum supremacy using Google's qsim simulator, and recreate their benchmarks and circuits.
+    :property="og:description": Use Google's qsim simulator to go beyond classical computing, and recreate their benchmarks and circuits.
```

Comment:
```suggestion
    :property="og:description": Use Google's qsim simulator to explore the barriers between quantum and classical computing, and recreate their benchmarks and circuits.
```
still think it sounds weird to use a classical computer program to go beyond classical computing :stuck_out_tongue_closed_eyes: